models/demos/t3000/falcon7b/README.md

```diff
@@ -1,43 +1,56 @@
-# Falcon7B Demo (T3000)
+# Falcon7B Demo 
```

Comment:
The "T3000" here is used to distinguish this README from the other Falcon7b demo READMEs (grayskull, wormhole, galaxy), I think it would be good to keep (t3000 can be changed to our external names for it as well).
Also it would be good to apply the formatting changes below to the other Falcon7b demo READMEs to keep them consistent.

Reply:
I see, I added back the T3000 delegation to the Falcon7b README.